#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_abstract_daemon_tmp_dir.dpatch by Javier Uruen Val <javi@warp.es>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to backport 0.11.100 fix to make abstract daemon use ebox tmp 
## DP: directory 

@DPATCH@

Index: src/EBox/AbstractDaemon.pm
===================================================================
--- libebox/src/EBox/AbstractDaemon.pm	(.../ubuntu-hardy-0.11.99/common/libebox)	(revisión: 9785)
+++ libebox/src/EBox/AbstractDaemon.pm	(.../ubuntu-0.11.99/common/libebox)	(revisión: 9785)
@@ -81,11 +81,11 @@
 
 	foreach my $fd (0 .. 64) { POSIX::close($fd); }
 
-
-	open(STDIN,  "+</tmp/stdin");
+	my $tmp = EBox::Config::tmp();
+	open(STDIN,  "+<$tmp/stdin");
 	if (EBox::Config::configkey('debug') eq 'yes') {
-	  open(STDOUT, "+>/tmp/stdout");
-	  open(STDERR, "+>/tmp/stderr");
+	  open(STDOUT, "+>$tmp/stout");
+	  open(STDERR, "+>$tmp/stderr");
 	}
